packages/core/contracts/merkle-distributor/implementation/MerkleDistributor.sol

```diff
@@ -103,14 +91,13 @@ contract MerkleDistributor is Ownable, Testable {
     //       their claims to suceed.
```

Comment:
Unrelated to this line, can we drop passing in timerAddress, inheriting from Testable, and the constructor entirely since we no longer have any time-dependent logic?

Reply:
yes! will do

Reply:
Weirdly removing this added 8 gas

Reply:
lol

Reply:
Have you tried nicely asking the solidity compiler to optimize harder?

Reply:
you think thats worth it? Its at 199 runs

Reply:
Haha, no -- just joking.